cmd/cli/completion.go

```diff
@@ -20,8 +20,9 @@ type oaiClients struct {
 func newOAIClients() (oaiClients, error) {
 	var config openai.ClientConfig
 	config = openai.DefaultConfig(*openAIAPIKey)
+	config.BaseURL = *openAIBase
 
-	if azureOpenAIEndpoint != nil || *azureOpenAIEndpoint != "" {
+	if *azureOpenAIEndpoint != "" {
```

Comment:
azureOpenAIEndpoint has a default value (empty string), so it cannot be nil. It will cause the if condition to always be true